tests/testthat/test-get_gsw_time_series.R

```diff
@@ -0,0 +1,34 @@
+test_that("get_gsw_time_series works", {
```

Comment:
Could this include a test to see that a footprints object is returned if successful?